instrumentation/hystrix-1.4/src/test/groovy/HystrixObservableTest.groovy

```diff
@@ -62,7 +62,6 @@ class HystrixObservableTest extends AgentTestRunner {
     }
 
     expect:
-    TRANSFORMED_CLASSES_NAMES.contains("HystrixObservableTest\$1")
```

Comment:
These are tested in the span name instead so I didn't bother exposing this through the agent classloader. It'd be simple enough though if we want to keep the functionality, these were the only tests using it though.